pylint/checkers/typecheck.py

```diff
@@ -1055,15 +1055,9 @@ def visit_attribute(self, node: nodes.Attribute) -> None:
                 return
 
             try:
-                if not [
-                    n
-                    for n in owner.getattr(node.attrname)
-                    if not isinstance(n.statement(future=True), nodes.AugAssign)
-                ]:
-                    missingattr.add((owner, name))
-                    continue
-            except astroid.exceptions.StatementMissing:
-                continue
+                attr_nodes = owner.getattr(node.attrname)
+            # pylint: disable-next=fixme
+            # TODO: remove except AttributeError when astroid updated to 2.10
             except AttributeError:
                 continue
```

Comment:
We will no longer need to catch `AttributeError` thanks to @areveny fixing it in https://github.com/PyCQA/astroid/pull/1366.

Test case:
```python
from astroid import extract_node
code = """
def f():
    pass
a = property(f)
a.fget
"""
node = extract_node(code)
prop = list(node.expr.infer())[0]
prop.getattr('fget')
```